core/integration/module_terminal_test.go

```diff
@@ -24,6 +24,75 @@ func TestModuleDaggerTerminal(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	t.Run("default arg /bin/sh", func(t *testing.T) {
```

Comment:
Do we need all of this test just to test for the panic case?

Reply:
Yes because this also test the default command if the user doesn't set anything (that we never tested before afaik)